### DISCUSSION TITLE
Preset led when using the Program change button

### DISCUSSION THREAD

**Discussion by js-midi** [shanteacontrols/OpenDeck#149] [Q&A]
Hello, thank you for allowing me to work on a great project.Let me ask you a question. Preset led reacts when the program change button is pressed.Is this how it's supposed to be? The firmware used is 7.0.0. Thank you.

**Comment by paradajz** [comment]
LEDs react on program change if you configure them to do so. Program change and preset change are separate settings.

**Reply by js-midi** [reply]
Thank you for your answer
